### DISCUSSION TITLE
add numbering into PDF's bookmark?

### DISCUSSION THREAD

**Discussion by retsyo** [typst/typst#1163] [Q&A]
Currently, the bookmark in generated PDF does not have numbering. Is there a way to turn it on? Thanks

![bug](https://github.com/typst/typst/assets/7960913/de668dd2-3ced-448e-93fa-2e4f063d8357)

**Comment by eaxca1** [comment]
I would like to use something like this as well. Have there been any updates regarding the inclusion of the heading numbering in the bookmarks? Is it possible to do it in any way? Thanks.

**Reply by laurmaedje** [reply]
It's not currently possible, but it shouldn't be too hard to add to the compiler.

**Reply by laurmaedje** [reply]
I've opened https://github.com/typst/typst/issues/2416 to track this and added the "good first issue" and "good contribution" labels. Maybe somebody will pick it up.

**Reply by eaxca1** [reply]
Alright, good to know, thanks!
